### PR TITLE
Resources: Android Projects added

### PR DESCRIPTION
**Pull request by GAVINESHWAR**
# Pull Request
issue : #1196 
## Description
Please include a summary of the changes and the related issue.  
Added the projects for the app development

Closes # 

## Type of change
- [X] Adding a new Resource
- [X] Adding a new Category
- [ ] Documentation enhancement or fixes
- [ ] Website enhancement or fixes

## Checklist:
- [X] I read carefully [CONTRIBUTING.md](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md)
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation

## Screenshots (if appropriate):

Please add screenshots to help explain your changes.
image
These are the github resources i am added .

## Additional 

I PR the issue check my changes and merge the PR @jfmartinz sir to make contribution to the project
Thank you

### CODE REVIEW

File: App Development/readme.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ App development refers to crafting software for diverse devices, involving plann
 - [YouTube Channels](#youtube-channels)<br>
 - [YouTube Playlists](#youtube-playlists)
 - [GitHub Repositories](#github-repositories)
+- [Projects](#github-repositories)
```
Comment (jfmartinz) [inline App Development/readme.md:15]:
```suggestion
- [Projects](#projects)
```

**Review by jfmartinz — CHANGES_REQUESTED** [review]
URLs that are not working:
1. Stopwatch
2. Voice to Text Convertor
3. To Do List
4. Dice a Roll

Don't forget to wrap the td with anchor tag to make the URLs work

File: App Development/readme.md (hunk 1 above)
Comment (GAVINESHWAR) [reply App Development/readme.md:15]: Sorry for trouble sir i resolved the issue u can merge the R @jfmartinz sir 
Than you

**Comment by GAVINESHWAR** [comment]
I Resolved the issue @jfmartinz now you can merge the changes and add  these resources sir, 
Tank you

**Review by jfmartinz — APPROVED** [review]

**Pull request merged**